web/ui/react-app/src/pages/targets/ScrapePoolList.tsx

```diff
@@ -25,7 +25,7 @@ export const ScrapePoolContent: FC<ScrapePoolListProps> = ({ activeTargets }) =>
       {}
     ),
   };
-  const [filter, setFilter] = useLocalStorage('targets-page-filter', initialState);
+  const [filter, setFilter] = useLocalStorage('targets-page-filter-expansion', initialState);
```

Comment:
What do we do next time? :)

Should we simply say -v2 ?

Reply:
I changed it to `-v2`. Is there a better way that involves invalidating the existing `localStorage` entry?

Reply:
Preferably we would have a smooth transition and handle the null in the current state?

Reply:
Yes, I'm just not sure of the best way to force the state/localStorage to update in the case that a key has a null value.

Reply:
i think based on [this comment](https://github.com/prometheus/prometheus/pull/8500#discussion_r579676051), the best solution is to create a new hook instead of modifying the old one. I will do that

Reply:
updated